run.py

```diff
@@ -739,7 +739,11 @@ def fetch_test_details(var) -> dict:
 
     for test in tests:
         test = test.get("test")
-        parallel = test.get("parallel")
+        try:
+            parallel = test.get("parallel")
+        except AttributeError:
+            log.error("Hint: Check your suite file for test definitions!")
+            raise
```

Comment:
Thanks... this allows us to raise an error gracefully and provide insights.

Unless `test` is None... we wouldn't get an attribute error. Can we do a check of `test` variable instead of proceeding.

Reply:
Can you please check now ?